HISTORY.rst

```diff
@@ -2,6 +2,27 @@
 History
 =======
 
+3.55.0 (2022-03-24)
+-------------------
+
+Changes
+
+- Tweaked Oauth App org restriction docs. Alters the trigger for warn_oauth_restricted error handler
+to reflect upstream changes. Also adds a note to the onboarding docs to make the workaround
+explicit. (#3044)
+- Usage examples for task reference, Adds example usage for a couple of tasks (#3110).
+- CumulusCI now provides an option to verify server certificates using root CA certs loaded from the system, instead of the default set of CA certs bundled with the Python requests library. This is considered experimental, so must be opted in by setting the CUMULUSCI_SYSTEM_CERTS environment variable to "True" (#3114)
```

Comment:
```suggestion
- CumulusCI now provides an option to verify server certificates using root CA certs loaded from the system, instead of the default set of CA certs bundled with the Python requests library. This is considered experimental, so must be opted in by setting the ``CUMULUSCI_SYSTEM_CERTS`` environment variable to ``True`` (#3114)
```